contents/tutorials/event-tracking-guide.md

```diff
@@ -2,184 +2,177 @@
+```
 
-**1. Through our toolbar (recommended for actions based on frontend elements)**
+Second, in whatever library you’ve chosen, you’ll need to configure PostHog. This includes adding your project’s API key and setting the PostHog host.
 
-To learn how to create actions using our toolbar, you can visit our [dedicated tutorial](/docs/tutorials/toolbar/#creating-actions).
+```python
+import posthog
 
-**2. From the events table**
+# Substitutes posthog.api_key which still exists but has been deprecated
+posthog.project_api_key = '<ph_project_api_key>'
 
-To create actions from the events table, just click on the plus sign on the left of any event to reveal a button allowing you to create an action from that specific event. You will also be able to customize the action further in the following steps (such as add another event). 
+# Only necessary if you want to use feature flags
+posthog.personal_api_key = '<ph_personal_api_key>'
 
-![Action from event image](../images/tutorials/actions/action-from-event.png)
-<small class='centered'>_Creating actions from the events table_</small>
+# You can remove this line if you're using app.posthog.com
+posthog.host = '<ph_instance_address>'
+```
 
+Third, once the library is installed and configured, events can be captured by calling library methods with user details, event types, and properties. Each of our libraries contains the relevant structure for connecting the arguments to these categories. For example, in Python a capture method call might look like this:
 
-**3. From the 'Actions' page**
+```python
+def movie_played(movie):
+	movie.view_count += 1
+	posthog.capture(
+		'distinct id', 
+		'movie played', 
+		{
+			'movie_id': movie.id,
+			'category': movie.category
+		}
+	)
+```
 
-Lastly, you can of course create actions from the 'Actions' page under 'Events':
+Adding more of these capture calls in the right places in your codebase creates a flow of event tracking data into PostHog. For more information on setting this up, read the **[live data ingestion guide](https://posthog.com/docs/integrate/ingest-live-data).**
 
-![Actions page image](../images/features/actions/actions-page.png)
+## Getting custom events right
 
-#### Action options
+With some custom events being sent, it is time to refine those events to capture the data you want. Getting this right requires multiple steps.
 
-Visit our [dedicated actions page](/docs/user-guides/actions/) for a comprehensive explanation of all the options available when creating an action.
+1. It starts with the goals of the product which inform what data is needed about users and their behavior. 
+2. Next is ensuring data is accessible in the right places in your codebase. This might require writing helper functions for access or formatting the data correctly.
+3. Finally, ensuring all the data is captured and added to PostHog for use in your analysis, visualization, and optimization. This is where we’ll focus for the rest of this tutorial.
```

Comment:
```suggestion
1. It starts with the goals of the product which inform what data is needed about users and their behavior.
 
2. Next is ensuring data is accessible in the right places in your codebase. This might require writing helper functions for access or formatting the data correctly.

3. Finally, ensuring all the data is captured and added to PostHog for use in your analysis, visualization, and optimization. This is where we’ll focus for the rest of this tutorial.
```
See previous